README.md

```diff
@@ -212,46 +212,40 @@ See [WebsocketClient](./src/WebsocketClient.ts) for further information and make
 
 ---
 
-## Customise Logging
+## Version Bumping Script
 
-Pass a custom logger which supports the log methods `trace`, `info` and `error`, or override methods from the default logger as desired.
+This script automatically bumps the version in `package.json` and `package-lock.json`.
 
-```javascript
-const { WebsocketClient, DefaultLogger } = require('kucoin-api');
-
-// E.g. customise logging for only the trace level:
-const logger = {
-  // Inherit existing logger methods, using an object spread
-  ...DefaultLogger,
-  // Define a custom trace function to override only that function
-  trace: (...params) => {
-    if (
-      [
-        'Sending ping',
-        // 'Sending upstream ws message: ',
-        'Received pong',
-      ].includes(params[0])
-    ) {
-      return;
-    }
-    console.log('trace', JSON.stringify(params, null, 2));
-  },
-};
-
-const ws = new WebsocketClient(
-  {
-    apiKey: 'apiKeyHere',
-    apiSecret: 'apiSecretHere',
-    apiPassphrase: 'apiPassPhraseHere',
-  },
-  logger,
-);
+### **How to Use**
+
+1. **Run the Script:**
+
+   ```bash
+   ./bump-version.sh <version>
+   ```
+
+   Replace `<version>` with the new version in `major.minor.patch` format (e.g., `1.0.1`).
+
+   If you don’t provide a version, it will prompt you to enter one.
+
+2. **What it Does:**
+   - Reads the current version from `package.json`.
+   - Updates the version in both `package.json` and `package-lock.json`.
+   - Commits the changes to Git with the message: `Bump Version to <new_version>`.
+
+3. **Dependencies:**
+   - This script uses `jq` to parse `package.json`. Install it using:
+     - **Ubuntu/Debian**: `sudo apt-get install jq`
+     - **macOS**: `brew install jq`
+
+#### **Example**
+
+```bash
+./bump-version.sh 1.0.1
```

Comment:
No need, see my other comment. Please revert this because you also accidentally deleted a few things from the readme.